packages/mui-joy/src/styles/defaultTheme.ts

```diff
@@ -1,165 +1,306 @@
+  md: CSSProperties['boxShadow'];
+  lg: CSSProperties['boxShadow'];
+  xl: CSSProperties['boxShadow'];
+}
+
+export interface Focus {
+  default: CSSObject;
 }
 
 export interface FontSize {
-  xs: React.CSSProperties['fontSize'];
-  sm: React.CSSProperties['fontSize'];
-  md: React.CSSProperties['fontSize'];
-  lg: React.CSSProperties['fontSize'];
-  xl: React.CSSProperties['fontSize'];
-  xl2: React.CSSProperties['fontSize'];
-  xl3: React.CSSProperties['fontSize'];
-  xl4: React.CSSProperties['fontSize'];
-  xl5: React.CSSProperties['fontSize'];
-  xl6: React.CSSProperties['fontSize'];
+  default: CSSProperties['fontSize'];
+  xs: CSSProperties['fontSize'];
+  sm: CSSProperties['fontSize'];
+  md: CSSProperties['fontSize'];
+  lg: CSSProperties['fontSize'];
+  xl: CSSProperties['fontSize'];
+  xl2: CSSProperties['fontSize'];
+  xl3: CSSProperties['fontSize'];
+  xl4: CSSProperties['fontSize'];
+  xl5: CSSProperties['fontSize'];
+  xl6: CSSProperties['fontSize'];
 }
 
 export interface FontFamily {
-  sans: React.CSSProperties['fontFamily'];
-  mono: React.CSSProperties['fontFamily'];
+  default: CSSProperties['fontFamily'];
+  display: CSSProperties['fontFamily'];
+  code: CSSProperties['fontFamily'];
+  fallback: CSSProperties['fontFamily'];
 }
 
 export interface FontWeight {
   // add string to support css variable value.
-  light: React.CSSProperties['fontWeight'] | string;
-  regular: React.CSSProperties['fontWeight'] | string;
-  medium: React.CSSProperties['fontWeight'] | string;
-  semiBold: React.CSSProperties['fontWeight'] | string;
-  bold: React.CSSProperties['fontWeight'] | string;
-  extraBold: React.CSSProperties['fontWeight'] | string;
-  black: React.CSSProperties['fontWeight'] | string;
+  default: CSSProperties['fontWeight'];
+  xs: CSSProperties['fontWeight'];
+  sm: CSSProperties['fontWeight'];
+  md: CSSProperties['fontWeight'];
+  lg: CSSProperties['fontWeight'];
+  xl: CSSProperties['fontWeight'];
 }
 
 export interface LineHeight {
-  xs: React.CSSProperties['lineHeight'];
-  sm: React.CSSProperties['lineHeight'];
-  normal: React.CSSProperties['lineHeight'];
-  lg: React.CSSProperties['lineHeight'];
+  default: CSSProperties['lineHeight'];
+  sm: CSSProperties['lineHeight'];
+  md: CSSProperties['lineHeight'];
+  lg: CSSProperties['lineHeight'];
 }
 
 export interface LetterSpacing {
-  xs: React.CSSProperties['letterSpacing'];
-  normal: React.CSSProperties['letterSpacing'];
-  lg: React.CSSProperties['letterSpacing'];
+  default: CSSProperties['letterSpacing'];
+  sm: CSSProperties['letterSpacing'];
+  md: CSSProperties['letterSpacing'];
+  lg: CSSProperties['letterSpacing'];
 }
 
-export interface Shadow {
-  ring: React.CSSProperties['boxShadow'];
-  md: React.CSSProperties['boxShadow'];
+export interface TypographySystem {
+  h1: CSSObject;
+  h2: CSSObject;
+  h3: CSSObject;
+  h4: CSSObject;
+  h5: CSSObject;
+  h6: CSSObject;
+  body1: CSSObject;
+  body2: CSSObject;
+  body3: CSSObject;
```

Comment:
This will probably change once we start designing more components and fewer marketing materials, is that ok?

Reply:
sure, we can change it later.